templates/page_header/page_header.component.inc

```diff
@@ -28,6 +33,11 @@ function europa_atomium_definition_page_header() {
     'description' => 'Page header.',
     'preview' => array(
       'markup' => '<p>The <strong>European Commission</strong> is the European Union’s executive body. It represents the interests of the European Union as a whole.</p>',
+      'header_bottom' => NULL,
+      'site_name' => NULL,
+      'page_title' => NULL,
+      'page_intro' => NULL,
+      'page_meta' => NULL,
     ),
```

Comment:
The definition of the component should have examples in them. Copying them from the ECL should be enough,